lib/src/util/dart_type_utilities.dart

```diff
@@ -176,12 +176,12 @@ class DartTypeUtilities {
 
   static bool isClass(DartType type, String className, String library) =>
       type != null &&
-      type.name == className &&
+      type.element?.name == className &&
       type.element?.library?.name == library;
 
   static bool isInterface(
```

Comment:
I see very little difference between this and the previous function.
I'd leave one, called `isInterfaceType()`.

Reply:
Good catch.